src/main/java/frc/robot/commands/balancing/DefaultBalancerCommand.java

```diff
@@ -16,18 +16,24 @@ public class DefaultBalancerCommand extends BaseBalancerCommand {
 
     private boolean reachedStation;
     private boolean passedCenter;
-    private boolean waiting;
     private boolean balanced;
 
     private double initialHeading;
     private double oldPitch;
     private double currentPitch;
     private double deltaAngle;
 
+    /**
+     * Constructs a default balancer command from a drive subsystem and a boolean indicating
+     * whether to balance from the front or back.
+     * 
+     * @param driveSubsystem The drive subsystem.
+     * @param reversed Whether to balance driving forwards or backwards. Defaults to backwards; pass `true` to balance forwards.
```

Comment:
The javadoc seems to have been left in; after this PR we can look into reversed balancing and maybe a better way of abstracting it?